shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.mm

```diff
@@ -27,41 +27,36 @@
     const fml::TimePoint target_time = recorder->GetVsyncTargetTime();
     FireCallback(start_time, target_time, true);
   };
-  client_ =
-      fml::scoped_nsobject{[[VSyncClient alloc] initWithTaskRunner:task_runners_.GetUITaskRunner()
-                                                          callback:callback]};
+  client_ = [[VSyncClient alloc] initWithTaskRunner:task_runners_.GetUITaskRunner()
+                                           callback:callback];
   max_refresh_rate_ = [DisplayLinkManager displayRefreshRate];
 }
 
 VsyncWaiterIOS::~VsyncWaiterIOS() {
   // This way, we will get no more callbacks from the display link that holds a weak (non-nilling)
   // reference to this C++ object.
-  [client_.get() invalidate];
+  [client_ invalidate];
 }
 
 void VsyncWaiterIOS::AwaitVSync() {
   double new_max_refresh_rate = [DisplayLinkManager displayRefreshRate];
   if (fabs(new_max_refresh_rate - max_refresh_rate_) > kRefreshRateDiffToIgnore) {
     max_refresh_rate_ = new_max_refresh_rate;
-    [client_.get() setMaxRefreshRate:max_refresh_rate_];
+    [client_ setMaxRefreshRate:max_refresh_rate_];
   }
-  [client_.get() await];
+  [client_ await];
 }
 
 // |VariableRefreshRateReporter|
 double VsyncWaiterIOS::GetRefreshRate() const {
-  return [client_.get() getRefreshRate];
-}
-
-fml::scoped_nsobject<VSyncClient> VsyncWaiterIOS::GetVsyncClient() const {
-  return client_;
+  return [client_ getRefreshRate];
 }
 
 }  // namespace flutter
 
 @implementation VSyncClient {
   flutter::VsyncWaiter::Callback callback_;
-  fml::scoped_nsobject<CADisplayLink> display_link_;
+  CADisplayLink* _displayLink;
   double current_refresh_rate_;
```

Comment:
Optional, but maybe fix the other two names while you are here so they aren't inconsistent?

Reply:
Done.